### PR TITLE
added modifier for api response format

### PR DESCRIPTION
**Pull request by RumyanaA**

### CODE REVIEW

File: client/src/app/discover/keyword-search/keyword-search.js [hunk 1]
```diff
@@ -14,40 +13,30 @@ function KeywordSearch() {
 
   const {
     recipes,
-    randomRecipes,
     hasResult,
     loading,
   } = useApi({ url });
 
+  useEffect(() => {
+    setUrl(`https://api.spoonacular.com/recipes/complexSearch?apiKey=${apiKey}&query=`);
```
Comment (bancho22) [inline client/src/app/discover/keyword-search/keyword-search.js:21]: I see no point in `usingEffect` to set the initial value of the url.
The same could be achieved by changing line 12 to:
`const [url, setUrl] = useState(`https://api.spoonacular.com/recipes/complexSearch?apiKey=${apiKey}&query=`);`

File: client/src/app/layout/main-content/shared-style.scss [hunk 2]
```diff
@@ -47,7 +47,7 @@
     }
 
     .recipe-cards-container{
-      width:100%;
+      width:97%;
```
Comment (bancho22) [inline client/src/app/layout/main-content/shared-style.scss:50]: 😢

File: client/src/app/discover/keyword-search/keyword-search.js (hunk 1 above)
Comment (RumyanaA) [reply client/src/app/discover/keyword-search/keyword-search.js]: fixed

File: client/src/app/layout/main-content/shared-style.scss (hunk 2 above)
Comment (RumyanaA) [reply client/src/app/layout/main-content/shared-style.scss:50]: It's prettier that way 🦓

Comment (bancho22) [reply client/src/app/layout/main-content/shared-style.scss:50]: Please try and make it prettier in a more stable way. What you're doing right now is an anti-pattern in CSS called "magic numbers".

Comment (RumyanaA) [reply client/src/app/layout/main-content/shared-style.scss]: https://github.com/RumyanaA/Recipe-finder/pull/21/commits/111c1f3e761bf7e5990b67acb789ce8ddabaaf7e

**Review by bancho22 — APPROVED** [review]

**Pull request merged**